src/common/intl/locales/fr.js

```diff
@@ -19,71 +19,150 @@ export default {
   confirmOk: 'OK',
   confirmCancel: 'Annuler',
 
-  // ----- General Menu -----
+  // ----- Filters -----
+  onError: 'En erreur',
+  successful: 'Réussi',
+
+  // ----- Copiable component -----
+  copyToClipboard: 'Copier dans le presse-papier',
+
+  // ----- Pills -----
+  pillMaster: 'Maître',
+
+  // ----- Titles -----
+  homePage: 'Accueil',
+  homeVmPage: 'VMs',
+  homeHostPage: 'Hôtes',
+  homePoolPage: 'Pools',
+  homeTemplatePage: 'Templates',
   dashboardPage: 'Tableau de bord',
   overviewDashboardPage: 'Vue d\'ensemble',
   overviewVisualizationDashboardPage: 'Visualisations',
   overviewStatsDashboardPage: 'Statistiques',
   overviewHealthDashboardPage: 'État de santé',
   selfServicePage: 'Self service',
-  selfServiceDashboardPage: 'Tableau de bord',
+  backupPage: 'Sauvegardes',
+  jobsPage: 'Jobs',
+  updatePage: 'Mises à jour',
+  settingsPage: 'Paramètres',
   settingsServersPage: 'Serveurs',
   settingsUsersPage: 'Utilisateurs',
   settingsGroupsPage: 'Groupes',
-  settingsAclsPage: 'ACLs',
-  settingsPluginsPage: 'Extensions',
-  selfServiceAdminPage: 'Administration',
-  backupPage: 'Sauvegarde',
-  backupOverviewPage: 'Vue d\'ensemble',
-  backupNewPage: 'Créer',
-  backupRemotesPage: 'Emplacement',
-  backupRestorePage: 'Restaurer',
-  updatePage: 'Mises à jour',
-  settingsPage: 'Paramètres',
+  settingsAclsPage: 'Droits',
+  settingsPluginsPage: 'Plugins',
```

Comment:
I think we want to have french people undertand XO, not having all the word in French.

French people are used to "Plugins", but they may not understand "Greffons".

Anyway, I can patch it if you think "Greffons" really are better.